packages/config/src/api/site_info.js

```diff
@@ -71,3 +79,25 @@ const getAddons = async function (api, siteId) {
     throwUserError(`Failed retrieving addons for site ${siteId}: ${error.message}. ${ERROR_CALL_TO_ACTION}`)
   }
 }
+
+const getIntegrations = async function ({ api, ownerType, ownerId, logs }) {
+  if (ownerId === undefined) {
+    return []
+  }
+
+  try {
+    const token = api.accessToken()
+    const response = await fetch(`https://api.netlifysdk.com/${ownerType}/${ownerId}/integrations`, {
```

Comment:
One thing to consider is that, unlike the regular Netlify API client, you don't have a retry mechanism in place here, so any transient failures in making the HTTP call would cause this to fail. I'm assuming that's okay for now given your comment in the `catch` block, but I thought I'd flag it anyway because this type of failure is something we do see a lot of in builds.

Reply:
Really appreciate the nuance on that, as you say for now it's no issue but in the future would be nice to gain some of the benefits